api.php

```diff
@@ -411,13 +425,12 @@
 	// if that id is not a number or is 0
 	if (!$id) {
 		// set a message to explain what broke
-		$response->getBody()->write(json_encode([
+		return badRequestResponse([
```

Comment:
I was able to create a new review that has a score greater than 5

Reply:
@margeschrec score is outside the scope of this ticket, please create the tickets for review routes with the required validation rules laid out.